forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt

```diff
@@ -0,0 +1,9 @@
+Name:Longhorn Sharpshooter
+ManaCost:2 R
+Types:Creature Minotaur Rogue
+PT:3/3
+K:Reach
+T:Mode$ BecomesPlotted | ValidCard$ Card.Self | TriggerZone$ Battlefield | Execute$ TrigDealDamage | TriggerDescription$ When CARDNAME becomes plotted, it deals 2 damage to any target.
```

Comment:
wrong TriggerZone

Reply:
your params still have a Typo
gotta test how this will play out

Reply:
The trigger was still "working" with either Zone... though I did notice one other glitch I was going to ask about.
Testing with right param now to see if the glitch gets solved

Reply:
Weird glitches:
 - if a trigger goes off, the plotted card stays face up (don't know if this is that big a deal as I'm not sure plotted cards should be turned face-down anyway?)
 - the trigger also seems to create a second "play for free" ability?
 
<img width="665" alt="image" src="https://github.com/Card-Forge/forge/assets/103371817/e6a53658-7719-4fc2-9901-04361e0d6b6c">

Reply:
just get rid of the facedown part